samples/cpp/text_generation/speculative_decoding_lm.cpp

```diff
@@ -25,6 +30,8 @@ int main(int argc, char* argv[]) try {
 
     // User can run main and draft model on different devices.
     // Please, set device for main model in `LLMPipeline` constructor and in in `ov::genai::draft_model` for draft.
+    // CPU, GPU and NPU can be used. Please be aware that GPU is performant only with Continuous Batching pipeline, so it is not recommented
+    // to use it in conjuction with NPU.
```

Comment:
I wouldn't put it this way here. Let's not make statements on other IPs with our changes.
I'd rather put it like for NPU, NPU+NPU configuration is preferrable - and that's it.